internal/integration_test/stdlibs/bench_test.go

```diff
@@ -0,0 +1,161 @@
+package wazevo_test
+
+import (
+	"context"
+	"crypto/rand"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+
+	"github.com/tetratelabs/wazero"
+	"github.com/tetratelabs/wazero/experimental/opt"
+	"github.com/tetratelabs/wazero/imports/wasi_snapshot_preview1"
+	"github.com/tetratelabs/wazero/internal/testing/require"
+	"github.com/tetratelabs/wazero/sys"
+)
+
+func BenchmarkStdlibs(b *testing.B) {
+	type testConfig struct {
+		name   string
+		config wazero.RuntimeConfig
+	}
+	configs := []testConfig{
+		{name: "baseline", config: wazero.NewRuntimeConfigCompiler()},
+	}
+	if runtime.GOARCH == "arm64" {
+		configs = append(configs, testConfig{name: "optimizing", config: opt.NewRuntimeConfigOptimizingCompiler()})
+	}
+
+	cwd, _ := os.Getwd()
+	defer os.Chdir(cwd) //nolint
+	ctx := context.Background()
+
+	testCases := []struct {
+		name, dir    string
+		readTestCase func(fpath string, fname string) ([]byte, wazero.ModuleConfig, error)
+	}{
+		{
+			name: "zig",
+			dir:  "testdata/zig/",
```

Comment:
if you use `go:embed` instead of reading dir, you should be able to build this benchmark as a single executable, hence no hiccups when running on windows/emulation?

Reply:
hiccups are more at run-time, e.g. the wasip1 executables expect to sit near some `testdata` dir in the source tree. Zig is probably the least problematic :(

Reply:
yep, Zig tests can be run as they are on Windows too. TinyGo and Wasip1 have issues with some paths. TinyGo I might be able to solve, but wasip1 is probably messier.